stories/modules-tagmanager-setup.stories.js

```diff
@@ -305,6 +349,13 @@ storiesOf( 'Tag Manager Module/Setup', module )
 							accountID,
 							internalContainerID,
 						} );
+
+					buildAndReceiveProperties(
+						registry,
+						accountID,
+						internalContainerID
+					);
+					provideWebContainer( registry, accountID );
```

Comment:
This call doesn't use any data from the parsed container version so it should go below (outside of this callback), same with other related calls.

Reply:
Good catch. Updated ✔️